Source/Core/DolphinQt/QtUtils/ModalMessageBox.cpp

```diff
@@ -6,13 +6,13 @@
 
 #include <QApplication>
 
-ModalMessageBox::ModalMessageBox(QWidget* parent) : QMessageBox(parent)
+ModalMessageBox::ModalMessageBox(QWidget* parent) : QMessageBox(parent->window())
```

Comment:
Is `parent->window()` a good idea, as you're checking `parent != nullptr` only a few lines after this?